src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable no-useless-constructor */
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  color: Color
+  shape: Shape
+  getArea(): number
+}
 
+function formatNumber(number: number): number {
+  return Math.floor(number * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = Shape.Triangle;
```

Comment:
Shape can be protected
```suggestion
  readonly shape = Shape.Triangle;
```